### DISCUSSION TITLE
Remove Alternate for Mobile

### DISCUSSION THREAD

**Discussion by scattagain** [ppy/osu#21196] [Ideas] [answered]
Alternate, Single tap, No scope and maybe some other mods are either Unfair or Impossible on mobile
Make these mods grayed out (Just like when they are incompatible with the currently chosen mods) on mobile

(Repel might also be a bit unfair on mobile)

**Comment by mk56-spn** [comment, marked as answer]
No scope is the only reasonable one to remove here. But considering it probably won't be giving extra score / pp ( due to just how many scenarios it can be cheesed in) I doubt it's a priority for the core Devs

As for single tap and alternate. In the final implementation of mobile touchscreen for the osu ruleset that is planned , it is my understanding that the tapping methods wouldn't differ much from PC osu ( youll be able to tap directly like tap X does on tablets , or use two fingers on the screen separately, which is analogous to tapping on a keyboard, like most common pc playstyles)

Aside from that the only major difference in the tap aspect would be mobiles lack of travel distance, seeing as mobile screens are capacitive 

But that's no different from this 

![148.jpg](https://user-images.githubusercontent.com/74463310/201072751-444bc375-a9b0-455a-add6-753f3821e69d.jpg)

**Reply by MasicoreLord** [reply]
Yeah, if no scope starts awarding extra score or pp at all, it should not be allowed for touch screen plays, or if so such scores should be unranked, as no scope on a touch screen is an unfair advantage if you get more score/pp, since the scope makes no difference there.

**Reply by bdach** [reply]
no scope will give exactly the same amount of pp as nomod
